blog/tests/admin/test_image_delete_fail.py

```
@@ -10,6 +10,16 @@
 from blog.models import PostImage
 
 
+@pytest.fixture(autouse=True)
+def aws_mock():
+    with mock_aws():
```

Ruff (0.15.6)

[error] Undefined name `mock_aws`

(F821)